wfe/wfe_test.go

```diff
@@ -693,7 +711,8 @@ func TestDirectory(t *testing.T) {
 	})
 	test.AssertEquals(t, responseWriter.Header().Get("Content-Type"), "application/json")
 	test.AssertEquals(t, responseWriter.Code, http.StatusOK)
-	assertJSONEquals(t, responseWriter.Body.String(), `{"key-change":"http://localhost:4300/acme/key-change","new-authz":"http://localhost:4300/acme/new-authz","new-cert":"http://localhost:4300/acme/new-cert","new-reg":"http://localhost:4300/acme/new-reg","revoke-cert":"http://localhost:4300/acme/revoke-cert"}`)
+	body := removeRandomKey(responseWriter.Body.Bytes())
```

Comment:
I'm not sure how to mock `core.RandomString`. So in order to get this PR started, I decided to remove the random entry from the response.

Reply:
I guess I could assert the json's values? Since the value of the random key is a constant.

Reply:
That's a good idea! You could have removeRandomKey return an error if randomDirKeyExplanationLink isn't found.

Reply:
Before I go ahead and change it to return an error, another approach would be to replace the random key with a fixed key, this way the assertions would be more expressive.

Reply:
Ah, that's also a good idea! Let's go with that approach.